src/platforms/javascript/common/install/cdn.mdx

```diff
@@ -51,7 +51,15 @@ Sentry.init({
 
 ## Available Bundles
 
-A variety of bundles are provided which are optimized for Sentry's various integrations.
+Our CDN hosts a variety of bundles:
+
+- `@sentry/browser` (named `bundle.<modifiers>.js`)
+- `@sentry/browser` and `@sentry/tracing` together (named `bundle.tracing.<modifiers>.js`)
+- each of the integrations in `@sentry/integrations` (named `<integration-name>.<modifiers>.js`)
+
+Each bundle is offered in both ES5 and ES6 versions, and for each version there are three bundle varieties: unminified, minified, and minified with debug logging. (That last version can be helpful for times when you need to debug an issue which only occurs in prod. In dev, it makes most sense to use the unminified bundle, which always includes logging.)
+
+So, for example, `bundle.tracing.es6.debug.min.js` is `@sentry/browser` and `@sentry/tracing` bundled together, compiled to ES6, and minified (but with debug logging included). On the simpler end of the naming spectrum, `bundle.js` is `@sentry/browser`, compiled to ES5 but not minified (with debug logging included, as it is for all unminified bundles).
```

Comment:
Once this is rendered, it's a bit hard to read, so suggesting the following formatting.

```suggestion
For example, `bundle.tracing.es6.debug.min.js` is:

- `@sentry/browser` and `@sentry/tracing` bundled together
- compiled to ES6
- minified, with debug logging included

A simpler example of this naming system would be `bundle.js`, which is:

- `@sentry/browser`
- compiled to ES5
- not minified, with debug logging included (as it is for all unminified bundles)

```